src/Grace/Repl.hs

```diff
@@ -0,0 +1,46 @@
+-- | This module contains the implementation of the @grace repl@ subcommand
+{-# LANGUAGE OverloadedStrings #-}
+
+module Grace.Repl
+    ( -- * REPL
+      repl
+    ) where
+
+import Control.Monad.IO.Class (liftIO, MonadIO)
+import Data.Text (pack)
+import Grace.Interpret (Input(..))
+
+import qualified Control.Monad.Except         as Except
+import qualified Data.Text.IO                 as Text.IO
+import qualified Grace.Interpret              as Interpret
+import qualified Grace.Normalize              as Normalize
+import qualified Grace.Pretty                 as Pretty
+import qualified System.Console.Repline       as Repline
+import qualified System.IO                    as IO
+
+repl :: IO ()
+repl =
+  Repline.evalRepl
```

Comment:
When I originally suggested using `evalRepl`, I didn't realize that it took the exact same options as `evalReplOpts`.  Given that, it might actually be clearer to use `evalReplOpts` because then all the arguments are named.  For example:

```haskell
Repline.evalReplOpts ReplOpts
    { banner = pure (pure ">>> ")
    , command = interpret
    , options = []
    …
    }
```